05/snake_for_if.py

```diff
@@ -0,0 +1,16 @@
+pets_start_second = []
+pets_sorted = []
+pets = ['had', 'andulka', 'pes', 'kočka', 'králík']
+copy_pets = pets
+
+for pet in pets:
+    pet = pet[0] + pet[1:]
+    pets_start_second.append(pet[1:])
+    pets_start_second.sort()
+for pet_second in pets_start_second:
```

Comment:
Ten zanorený for cyklus je taký kostrbatý. Iteruješ dohromady 25x, pričom by vlastne stačilo len 5x ak by si si to nezoradila pred tým a spojila k sebe `animals` a `pets_start_second`. Na hodine sme si ukazovali funckiu `zip`, tá je na to šikovná. Potom by ti už stačilo si celý tento zoznam dvojíc zoradiť a vytiahnuť si z nich do nového zoznamu zoradené zvieratká.